api/src/config.rs

```diff
@@ -875,7 +875,13 @@ pub struct PrefetchConfigV2 {
     #[serde(default = "default_prefetch_threads")]
     pub threads: usize,
     /// The batch size to prefetch data from backend.
-    #[serde(default = "default_prefetch_batch_size")]
+    /// A.k.a. [PrefetchConfigV2::batch_size],
+    /// [FsPrefetchControl::merging_size],
+    /// [BlobPrefetchConfig::merging_size],
+    /// [AsyncPrefetchConfig::merging_size],
+    /// [default_prefetch_request_batch_size()],
+    /// [FileCacheEntry::prefetch_request_batch_size()].
+    #[serde(default = "default_prefetch_request_batch_size")]
```

Comment:
It means changes for user. Please update [doc](https://github.com/dragonflyoss/nydus/blob/master/docs/nydusd.md) + add [smoke test](https://github.com/dragonflyoss/nydus/blob/master/tests/smoke.rs) and unit tests

Reply:
This setting is used to set the default value function name, and does not change the variable names in the config file. So there's no compatibility issue.